tests/Tests/100__deploy/130__operators/130__rhods_operator/135__rhods_operator_logs_verification.robot

```diff
@@ -23,7 +23,7 @@ Verify RHODS Operator log
    #Capture the logs based on containers
    ${val}        Run   oc logs --tail=1000000 ${data[0]['metadata']['name']} -n ${namespace} -c rhods-operator
    #To check if command has been suessfully executed and the logs has been captured
-   Run Keyword IF    len($val)==${0} or "error" in $val     FAIL   Either OC command has not been executed suessfully or Logs is not present
+   Run Keyword IF    len($val)==${0} or "error:" in $val     FAIL   Either OC command has not been executed suessfully or Logs is not present
```

Comment:
is it a good idea to add regex example "error*" so that in the future we may not face the same issue

Reply:
i don't think so...i we use error or error* we will be capturing every error present in the sentences as well  which might be not be the actual error. the idea to have here is that when we run oc command it succeed without any error. When we use wrong command we get error like below:

ex: error:*